src/store/asana/index.ts

```diff
@@ -575,6 +583,10 @@ async function completeTask(tasks: Task[], gid: string) {
   });
 }
 
+async function getTaskAttachments(gid: string)  {
+  return asanaClient?.attachments.findByTask(gid, {opt_fields: "view_url, permanent_url, name"});
```

Comment:
Shouldn't need nullable op on asanaClient

Reply:
The definition of asanaClient allows it to be null. The compiler complains if I take out the ?

Reply:
Either should be a ! operator (to assert not null) or a check and error if null. Right now, it not being null won't trigger any indication of that state (which shouldn't be allowed for this function)